include/sharg/parser.hpp

```diff
@@ -158,11 +158,11 @@ class parser
     /*!\name Constructors, destructor and assignment
      * \{
      */
-    parser() = delete;                            //!< Deleted.
-    parser(parser const &) = default;             //!< Defaulted.
-    parser & operator=(parser const &) = default; //!< Defaulted.
-    parser(parser &&) = default;                  //!< Defaulted.
-    parser & operator=(parser &&) = default;      //!< Defaulted.
+    parser() = delete;                           //!< Deleted.
+    parser(parser const &) = delete;             //!< Deleted.
+    parser & operator=(parser const &) = delete; //!< Deleted.
```

Comment:
Can you please add a changelog for these under notable bug fixes (as I assume it is no API break because copying wasn't possible anyway?)

Also can you add a note in the documentation (instead of`Deleted`) as of why they are deleted.

Reply:
No, not really, it is pure annotation error of the source code. It has no meaning.

Also I really don't want to add any documentation why they are deleted. It is just there, for doxygen and clarity. Adding documentation is to much overhead.

Reply:
After talking to @SGSSGene in person I agreed that this isn't helpful to the user because `sub_parser` is something internal and internal workings shouldn't be exposed as they usually just confuse the user. As there is no valuable explanation, just stating deleted will suffice.

Reply:
Note that an explicit explanation is given in the commit message.